test/redditor/redditor_test.dart

```diff
@@ -372,4 +372,11 @@ Future<void> main() async {
     expect(content[0] is Submission, isTrue);
     expect((content[0] as Submission).title, "Gilded post!");
   }, skip: 'Rewrite to handle new gilding types');
+
+  test('lib/redditor/trophies', () async {
+    final reddit = await createRedditTestInstance(
+        'test/redditor/lib_redditor_trophies.json');
+    final trophies = await reddit.redditor('spez').trophies();
+    expect(trophies, isNotEmpty);
```

Comment:
Thanks for writing a test! Would you mind doing some simple checks for the fields of a single `Trophy` just as a sanity check?